File: UI Automation C#/Exercise MsTest V2/Exercise MsTest/Exercise MsTest/Base/UnoBrowser.cs
using OpenQA.Selenium;
using OpenQA.Selenium.Chrome;
using OpenQA.Selenium.Support.UI;
using SeleniumExtras.WaitHelpers;
using System;
using System.Collections.Generic;
using System.IO;
using System.Reflection;
using System.Text;

namespace Exercise_MsTest.Base
{
    public class UnoBrowser
    {
        public enum Browser
        {
            Chrome
        }
        IWebDriver driver;
        public IWebDriver CreateBrowser(Browser browser)
        {
            //IWebDriver driver = new ChromeDriver(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location));
            driver = new ChromeDriver(@"C:\Users\heriberto.vasquez\Documents\WebDrivers\Chrome"); //local webdriver
            driver.Manage().Window.Maximize();
            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(6);
            driver.Navigate().GoToUrl("https://www.unosquare.com");
            return driver;
        }

        public void Click(IWebElement element)
        {
            element.Click();
        }

        public void SendText(IWebElement element, string value)
        {
            element.SendKeys(value);
        }

        public void explicitWait(int seconds)
        {
            WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(seconds));
        }

        //This method wait the element until it exists
        public void waitForElement(int seconds, By xpath)
        {
            WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(seconds));
            wait.Until(OpenQA.Selenium.Support.UI.ExpectedConditions.ElementExists(xpath));
        }
    }
}

File: UI Automation Java/Exercise 3 - Automation Java/automation_3/src/main/java/Base/UnoBrowser.java
package Base;

import java.util.concurrent.TimeUnit;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

import automation_TestNG.BrowserInteraction;

public class UnoBrowser extends BrowserInteraction {
	WebDriver driver;
	public WebDriver CreateBrowser() {
		System.setProperty("webdriver.chrome.driver", "C:\\Users\\heriberto.vasquez\\Documents\\WebDrivers\\Chrome\\chromedriver.exe");
		driver = new ChromeDriver();
		driver.manage().window().maximize();
		driver.get("https://www.amazon.com.mx/");
		driver.manage().timeouts().implicitlyWait(15, TimeUnit.SECONDS);
		return driver;
	}

}
